02.redis/Dockerfile

```diff
@@ -14,4 +14,4 @@ ADD Gemfile /app/Gemfile
 ADD Gemfile.lock /app/Gemfile.lock
 RUN bundle install --jobs 20 --retry 5
 
-Add . /app
+ADD . /app
```

Comment:
Should we switch all `ADD` statements to `COPY`, since this is recommended by Docker unless you require the special `ADD` features?